src/test/java/ui/CatalogueTest.java

```diff
@@ -1,19 +1,28 @@
 package ui;
 
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 import pages.CataloguePage;
+import pages.MainPage;
 import ui.driverSetup.BaseTestSetup;
+import util.dataUtils.CharDataForTestSite;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class CatalogueTest extends BaseTestSetup {
 
     private CataloguePage cataloguePage;
 
-    @Test
-    public void openCataloguePageFromMainPage() throws InterruptedException {
-        mainPage.openAnyCataloguePage();
+    @BeforeMethod
+    public void catTestSetUp() {
+        driver.get(CharDataForTestSite.HOME_URL);
```

Comment:
Should be encapsulated in `.openPage()` method. Can be refactored in further PRs